samples/powerpoint/preview-apis/tags.yaml

```
@@ -0,0 +1,219 @@
+
+            await context.sync();
+
+            console.log("Added key " + JSON.stringify(myShapeTag.key) + " with value " + JSON.stringify(myShapeTag.value));
+          });
+        }
+
+        async function addPresentationTag() {
+          await PowerPoint.run(async function (context) {
+            let presentationTags = context.presentation.tags;
+            presentationTags.add("COLOR", "blue");
+
+            await context.sync();
+
+            const tag = presentationTags.getItem("COLOR");
+            tag.load("key, value");
+
+            await context.sync();
+
+            console.log("Added key " + JSON.stringify(tag.key) + " with value " + JSON.stringify(tag.value));
+          });
+        }
+
+        async function deletePresentationTag() {
+          await PowerPoint.run(async function (context) {
+            let presentationTags = context.presentation.tags;
+
+            presentationTags.delete("COLOR");
+
+            await context.sync();
+
+            console.log(JSON.stringify(presentationTags));
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>These snippets show how to use tags with the presentation and its slides and shapes.</p>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+
+            <p>1. Add several slides to the deck. Add content to each so they are visually distinct in the thumbnail pane.</p>
+
+            <p>2. Select a <i>single</i> slide and press <b>Add tag</b> to tag the slide to be show only to premium customers.</p>
+            <button id="add-selected-slide-tag" class="ms-Button">
+                <span class="ms-Button-label">Add tag</span>
+            </button>
+
+            <p>3. Repeat step 2 for another slide.
+
+            <p>4. Press <b>Delete premium customer slides</b> to remove from the presentation slides that should only be shown to premium customers.</p>
+            <button id="delete-slides-by-audience" class="ms-Button">
+                <span class="ms-Button-label">Delete premium customer slides</span>
+            </button>
+
+            <p>5. Press <b>Add slide tags</b> to add mulitiple tags to the first slide of the presentation.</p>
+            <button id="add-slide-tags" class="ms-Button">
+                <span class="ms-Button-label">Add slide tags</span>
+            </button>
+
+            <p>6. Select the first slide and on the ribbon, navigate <b>Insert</b> | <b>Illustrations</b> | <b>Shapes</b> to add a shape to it. Press <b>Add shape tag</b> .</p>
```

Maybe switch from `|` to `>` (may need to encode `>`: `&gt;`)
```suggestion
            <p>6. Select the first slide and on the ribbon, navigate <b>Insert</b> > <b>Illustrations</b> > <b>Shapes</b> to add a shape to it. Press <b>Add shape tag</b>.</p>
```